import-export-cli/impl/importAPI.go

```diff
@@ -950,3 +494,129 @@ func ImportAPI(accessOAuthToken, adminEndpoint, importEnvironment, importPath, a
 	err = importAPI(adminEndpoint, apiFilePath, accessOAuthToken, extraParams)
 	return err
 }
+
+// envParamsFileProcess function is used to process the environment parameters when they are provided as a file
+func envParamsFileProcess(importPath, paramsPath, importEnvironment string) error {
+	apiParams, err := params.LoadApiParamsFromFile(paramsPath)
+	if err != nil {
+		return err
+	}
+	// check whether import environment is included in api configuration
+	envParams := apiParams.GetEnv(importEnvironment)
+	if envParams == nil {
+		utils.Logln(utils.LogPrefixInfo + "Using default values as the environment is not present in api_param.yaml file")
```

Comment:
In the previous approach, it does not need to return an error as we only follow this path when api_params.yaml is provided. This checks whether the provided api_params.yaml file contains the runtime environment or not. It is possible to provide an external api_params file without the current runtime environment.

But with the new approach when an external api_params.yaml file is provided, we have to throw an error since the source API project does not contain a default file. Thanks for pointing this out.